src/economy/portdock.cc

```diff
@@ -611,9 +510,6 @@ void PortDock::Loader::load_pointers() {
 	PortDock& pd = get<PortDock>();
 	pd.warehouse_ = &mol().get<Warehouse>(warehouse_);
 
-	for (Serial s : ships_coming_) {
```

Comment:
Comment for load_pointers:
// Restore connections between objects after load -- or better